modules/auxiliary/gather/pimcore_creds_sqli.rb

```diff
@@ -35,7 +35,7 @@ def initialize(info = {})
         {
           'SideEffects' => [ IOC_IN_LOGS ]
         },
-      'DisclosureDate' => 'Aug 13, 2018'
+      'DisclosureDate' => 'Aug 13 2018'
```

Comment:
This PR just removes the miscellaneous punctuation, a future PR will align the DisclosureDates to be formatted as ISO8601.